### ISSUE TITLE
Allow to slide through years in the final graphic.

### ISSUE THREAD

**Issue by l00mi** [visualize-admin/visualization-tool#451]
**Is your feature request related to a problem? Please describe.**

To inspect the development of a dataset in time is difficult if the time line is not part of the X or Y axis.

You need to select each year manually. With a slider we could provide instantly something similar to https://www.youtube.com/watch?v=hVimVzgtD6w which is very nice to get a "feel" data for all types of graphics.

**Describe the solution you'd like**

Provide a Slider for Continuous dimension (like e.g. YEAR) in the final graphic filter.
![Screenshot 2022-03-14 at 16 54 30](https://user-images.githubusercontent.com/502518/158211244-7fc3d743-96ab-4616-b14b-9bd1330b685e.png)

* Probably we could even add a play button, and a looping feature.
* It might be only be use-full for "time" dimensions, but we should see how it works with other continuous dimensions too.

**Describe alternatives you've considered**
My proposition is to provide it where we have filters already, but it might be worth to have a special "filter" which is always visible, (not only after opening "show filter").

**Usecases and impact**

All graphics will immediately profit from the use.

**Comment by AFoletti** [comment]
Similar to https://github.com/visualize-admin/visualization-tool/issues/444, if I may add 😄

**Comment by Rdataflow** [comment]
very interesting 
@Rdataflow subscribes

**Comment by bprusinowski** [comment]
Should be closed when we implement time slider feature (starting with #834).

**Comment by bprusinowski** [comment]
Closed by the latest work on animations (#1101, #1103, #1105).

**Comment by l00mi** [comment]
🎉
